### DISCUSSION TITLE
Better documentation.

### DISCUSSION THREAD

**Discussion by pudrik** [primefaces/community#59] [PrimeNG] [answered]
Hi ive noticed that in every new version of primeng the documentation seems to get poorer and poorer.

To name one thing that get old very fast is going to the documentation opens the latest version. going to the component to find something is missing or not documented, then going to an old version and then clicking my way to the same info, how about altering the behavior so it doesnt go to the version startpage but instead goes to the same documentation but a different version?

Also there are alot of gaps in the latest documentation for instance menu models are missing.

Alot of the components lack examples, kinda hard to know how it was intended to be used when there is 0 reference code.

Some components arent even documented like p-overlay

anyway without going into every single detail ive noticed it might be worth taking a second look at the documentation and update it accordingly wouldnt you agree ?

**Comment by cetincakiroglu** [comment, marked as answer]
Hello,

As of 2023, we have updated the documentation for all our Prime UI Libraries. Compared to their previous versions, we've adopted a design that showcases the individual features of each component, and we've included StackBlitz examples for these features in the documentation. We did this to provide our valued users with a user-friendly documentation that allows them to easily review the features. For easier discovery of components and their features, we added a search functionality with Algolia search.

Some components have been deprecated and are naturally not included in the documentation for v16. Additionally, information related to the Menu Model can be accessed via Menu > Api > MenuItem. All menu components have a MenuItem in their API section. An example link is here: https://primeng.org/menu#api.menu.interfaces.MenuItem

In v15, the p-overlay component has been added to the documentation. However, since this component is generic and serves as a wrapper, it has been added to the utilities section rather than the components section. We appreciate you reporting its absence from the components menu in v16, and we're addressing the correction. You can still reach out to the documentation of p-overlay from the URL https://primeng.org/overlay.

The discrepancy between the old and new websites is noted to be fixed in the future. Thank you for reporting the issue; we'll work on it.

Thanks for your feedback

**Comment by cagataycivici** [comment]
On the contrary, I think latest docs are better than ever, the community feedback has been great.
